bin/codecept.js

```diff
@@ -158,16 +158,35 @@ program.command('run-multiple [suites...]')
 
 program.command('info')
   .description('Print debugging information concerning the local environment')
-  .action(() => {
+  .action(async () => {
+    const info = {};
     console.log('\n Environment information:-\n');
-    envinfo
-      .run({
-        System: ['OS', 'CPU'],
-        Binaries: ['Node', 'Yarn', 'npm'],
-        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
-        npmGlobalPackages: ['codeceptjs'],
-      })
-      .then(console.log);
+    info.codeceptVersion = Codecept.version();
+    info.nodeInfo = await envinfo.helpers.getNodeInfo();
+    info.osInfo = await envinfo.helpers.getOSInfo();
+    info.cpuInfo = await envinfo.helpers.getCPUInfo();
+    info.chromeInfo = await envinfo.helpers.getChromeInfo();
+    info.edgeInfo = await envinfo.helpers.getEdgeInfo();
+    info.firefoxInfo = await envinfo.helpers.getFirefoxInfo();
+    info.safariInfo = await envinfo.helpers.getSafariInfo();
+    const fs = require('fs');
+    const config_path = `${process.cwd()}/codecept.conf.js`;
+    if (fs.existsSync(config_path)) {
+      const { helpers, plugins } = require(`${process.cwd()}/codecept.conf.js`).config;
```

Comment:
Please, load config the same way as in other scripts. See 

https://github.com/Codeception/CodeceptJS/blob/master/lib/command/dryRun.js#L15